lib/mss/mss_parser.js

```diff
@@ -250,7 +248,6 @@ shaka.mss.MssParser = class {
     const updateDuration = (endTime - startTime) / 1000.0;
     this.averageUpdateDuration_.sample(1, updateDuration);
 
-    // Let the caller know how long this update took.
     return updateDuration;
```

Comment:
When I said this didn't seem necessary, I was referring to the return value, not just the comment.  No caller of this seems to use the return value, which is tracked internally in `averageUpdateDuration_` instead.

Reply:
Sorry, changed, I misunderstood you :(